FILE packages/cli/package.json
RANGE {"path":"packages/cli/package.json","old_lines":[23,28],"new_lines":[23,28],"diff_block_lines":[12,20],"hunk_header":"@@ -23,6 +23,6 @@","substantive_changed_lines":4}
@@ -23,6 +23,6 @@
   "dependencies": {
     "@clack/prompts": "^1.0.0",
-    "@getpaseo/relay": "0.1.14",
-    "@getpaseo/server": "0.1.14",
+    "@getpaseo/relay": "0.1.15",
+    "@getpaseo/server": "0.1.15",
     "chalk": "^5.3.0",
     "commander": "^12.0.0",
